packages/core/directives/marker.ts

```diff
@@ -61,7 +62,7 @@ export class AgmMarker implements OnDestroy, OnChanges, AfterContentInit {
   /**
    * The label (a single uppercase character) for the marker.
    */
-  @Input() label: string;
+  @Input() label: string | MarkerLabel;
```

Comment:
we should add this type to: https://github.com/SebastianM/angular-google-maps/blob/master/packages/core/map-types.ts

Reply:
did that, waiting for review